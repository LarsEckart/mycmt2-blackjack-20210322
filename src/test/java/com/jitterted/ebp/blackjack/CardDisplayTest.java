package com.jitterted.ebp.blackjack;

import org.approvaltests.Approvals;
import org.junit.jupiter.api.Test;

class CardDisplayTest {

  @Test
  void displayTenAsString() {
    Card card = new Card(Suit.HEARTS, Rank.TEN);

    Approvals.verify(card.display());
  }

  @Test
  void displayNonTenAsString() {
    Card card = new Card(Suit.CLUBS, Rank.SEVEN);

    Approvals.verify(card.display());
  }
}
