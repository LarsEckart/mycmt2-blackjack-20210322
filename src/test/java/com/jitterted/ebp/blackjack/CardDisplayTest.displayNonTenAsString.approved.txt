[30m┌─────────┐[1B[11D│7        │[1B[11D│         │[1B[11D│    ♣    │[1B[11D│         │[1B[11D│        7│[1B[11D└─────────┘